.github/workflows/build.yaml

```diff
@@ -0,0 +1,49 @@
+name: Build
+on:
+  pull_request:
+    branches:
+    - master
+  push:
+    branches:
+    - master
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-20.04
+    env:
+      IMAGE_NAME: whatwg/html-build
+    steps:
+    - name: Checkout whatwg/html-build
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
```

Comment:
Why do we need a full checkout? If it's just because of the up-to-date check, maybe we could skip that with a command line arg?

Reply:
It's so we can tag it with the revision number. This is a bit silly, indeed. Perhaps we should tag it with the checkout SHA instead?

Reply:
Oh right. Well, I guess this repo isn't so big that it's a big problem. Tagging with the commit hash would be fine too, but no need to change it if you don't feel like it.